Ports/halflife/package.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port="halflife"
+version="1.0.0"
+useconfigure="true"
+depends=("SDL2")
+workdir="xash3d-fwgs-master"
+files="https://github.com/SerenityPorts/xash3d-fwgs/archive/master.tar.gz xash3d_engine.tar.gz
+https://github.com/SerenityPorts/hlsdk-xash3d/archive/master.tar.gz xash3d_hldll.tar.gz"
+launcher_name="Half-Life"
+launcher_category="Games"
+launcher_command="sh /home/anon/Games/halflife/hl.sh"
+
+# This one is a bit tricky to build, so I'm going a little bit off the script....
+configure() {
+    # Configure the shared object projects (client and game)
+    cd ./hlsdk-xash3d-master
+    ./waf configure -T release
+    cd ../
+
+    # Configure the engine itself...
+    cd ./xash3d-fwgs-master
+    ./waf configure --sdl2=${SERENITY_INSTALL_ROOT}/usr/local -T release
+}
+
+build() {
+    # Build the game and client
+    cd ../hlsdk-xash3d-master
+    ./waf build
+    cd ../
+
+    # Build the engine
+    cd ./xash3d-fwgs-master
+   ./waf build
+    cd ../
+}
+
+install() {
+    cd ./hlsdk-xash3d-master
+    ./waf install --destdir=${SERENITY_INSTALL_ROOT}/home/anon/Games/halflife
+    cd ../
+
+    # Build the engine
+    cd ./xash3d-fwgs-master
+    ./waf install --destdir=${SERENITY_INSTALL_ROOT}/home/anon/Games/halflife/
+    cd ../
+}
+
+post_install() {
+    # Delete the OpenGL renderer (ref_gl.so) so we don't crash on launch...
+    rm ${SERENITY_INSTALL_ROOT}/home/anon/Games/halflife/libref_gl.so
+    
+    # Rename the installed library files
+    mv ${SERENITY_INSTALL_ROOT}/home/anon/Games/halflife/valve/dlls/libhl.so ${SERENITY_INSTALL_ROOT}/home/anon/Games/halflife/valve/dlls/hl_serenityos_i386.so
+    mv ${SERENITY_INSTALL_ROOT}/home/anon/Games/halflife/valve/cl_dlls/libclient.so ${SERENITY_INSTALL_ROOT}/home/anon/Games/halflife/valve/cl_dlls/client_serenityos_i386.so
```

Comment:
Does this work on x86_64?!

Reply:
Not sure, I'll take it for a spin and replace this with `${SERENITY_ARCH}` if it does.

Reply:
Aaaaaand it does! Updated to reflect this change :^)